internal/pkg/httpserver/server.go

```diff
@@ -0,0 +1,225 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package httpserver
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+	"sync/atomic"
+	"time"
+
+	"github.com/gorilla/mux"
```

Comment:
you can create new instance of metric provide server in cmd and use eco server

Reply:
using echo now